pkg/executor/internal/exec/executor.go

```diff
@@ -89,6 +90,9 @@ func newExecutorChunkAllocator(vars *variable.SessionVars, retFieldTypes []*type
 
 // InitCap returns the initial capacity for chunk
 func (e *executorChunkAllocator) InitCap() int {
+	failpoint.Inject("initCap", func (val failpoint.Value) {
```

Comment:
Dose it mean the chunk size would be a constant value If the `initCap` is set to a value same with `maxChunkSize` using both two failpoints?

Reply:
Yes that's my expectation.